pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -156,11 +157,15 @@ class ModelCheckpoint(Callback):
         filepath: string, path to save the model file.
         monitor: quantity to monitor.
         verbose: verbosity mode, 0 or 1.
-        save_best_only: if `save_best_only=True`,
+        save_best: if `save_best=True`,
```

Comment:
this makes the parameter list very long, I would suggest using something like `saving_mode=('best', 'all')`

Reply:
Good idea! 👍